frontend/src/metabase/lib/errors/validation.ts

```diff
@@ -2,14 +2,14 @@ import { t } from "ttag";
 
 import type { LengthParams, MaxLengthParams } from "./types";
 
-export const required = () => t`required`;
+export const required = () => t`Required`;
 
-export const email = () => t`must be a valid email address`;
+export const email = () => t`Must be a valid email address`;
 
 export const maxLength = ({ max }: MaxLengthParams) =>
-  t`must be ${max} characters or less`;
+  t`Must be ${max} characters or less`;
```

Comment:
A little context would be helpful here, along with similar context for any variables occurring in translated strings, if you think the grammar of the variable might not be easy for the translators to figure out

```suggestion
  c("{0} is a number").t`Must be ${max} characters or less`;
```